tests/AsyncRgbLedTestDriver.cpp

```diff
@@ -106,7 +107,7 @@ class LedChannelDataGenerator
               range = (t.max - t.min) * mTolerance;
 
         // we want to offset by a random amount of the range
-        double offsetRandom = drand48() - 0.5; // random is now -0.5 .. 0.5
+        double offsetRandom = rand() - 0.5; // random is now -0.5 .. 0.5
```

Comment:
`drand48()` produced a value from [0.0 to 1.0), where `rand()` produces a value from 0 to RAND_MAX.